src/v2/view-builder/views/ov/EnrollPollOktaVerifyView.js

```diff
@@ -71,6 +67,16 @@ const Body = BaseForm.extend(Object.assign(
 
       }
     },
+    postRender() {
+      // Using setTimeout of 0 because, the view is added in postRender of formController and not the initialize,
+      // and hence by the time swithForm is called, current view is not rendered and availabe on controller $el.
+      setTimeout(() => {
+        if ((BrowserFeatures.isAndroid() || BrowserFeatures.isIOS()) &
+          this.options.appState.get('currentAuthenticator').contextualData.selectedChannel === 'qrcode') {
+          this.options.appState.trigger('switchForm', RemediationForms.SELECT_ENROLLMENT_CHANNEL);
+        }
+      }, 0);
+    },
     getUISchema() {
```

Comment:
should this return empty if we are in Android or iOs? Since we don't really need to render the QR code

Reply:
@mauriciocastillosilva-okta  turns out the change that overrides the basecontroller render caused the issue. once that was reverted the issue got resolved. I created a ticket to just add a test to catch the regression in future. 
https://github.com/okta/okta-signin-widget/commit/018a85a76ce6c5ec270a333e6d6d028ec8183b6f#diff-563f8e94ab62f12655040f13d4aa05bf9111fecde04b01c9f39d29c2ff03467d